docker-compose.yml

```diff
@@ -150,3 +172,17 @@ centos7test:
     service: volumes-compose
   volumes:
     - /sys/fs/cgroup:/sys/fs/cgroup
+
+centos8test:
+  image: packagingtest:centos8-systemd
```

Comment:
I'll need to rewind to https://github.com/StackStorm/st2packaging-dockerfiles/pull/74 PR first so we can upload those images to Docker Hub.

Reply:
@armab I have updated StackStorm/st2packaging-dockerfiles#74